app/git/utils.py

```diff
@@ -629,8 +762,7 @@ def post_issue_comment_reaction(owner, repo, comment_id, content):
     """React to an issue comment."""
     gh_client = github_connect()
     try:
-        repo = gh_client.get_repo(f'{owner}/{repo}')
-        reaction = repo.get_comment(comment_id).create_reaction(content)
+        reaction = _get_commit_comment(gh_client, owner, repo, comment_id).create_reaction(content)
         return reaction
```

Comment:
@thelostone-mc @gdixon Same issue as above

Reply:
Hmm it seems like an overkill! I'd be in the favour of removing it